### PR TITLE
Feature/filter favorites

### PR DESCRIPTION
**Pull request by Echory**
## Checklist
- [ ] Tests written where required
- [ ] All tests passing
- [ ] Unused code/prys/etc deleted
Testing Coverage at <percentage>%

## Changes Made
  
What (if any) features are you implementing?
- Implemented a filter to view only favorites or all submissions. Users can also unfavorite.

Were there any issues that arose?
- Had some issues with state but Ryan helped me figure it out!

Any other comments, questions, or concerns?
- Thanks for helping me with this Ryan, getting the data with GraphQL can be a bit tricky

### CODE REVIEW

File: src/components/AdminPage/AdminPage.js [hunk 1]
```diff
@@ -5,30 +5,51 @@ import { useQuery } from '@apollo/client';
...
+    const filteredSubmissions = data.getSubmissions.filter(submission => submission.adminFavorite).map(submission => {
+      return <MusicianCard refetch={refetch} user={user} key={submission.id} submission={submission}/>
+    })
+
+    const showFilter = user && <div>
+      <label>View All</label>
+      <input type="radio" name="filter" defaultChecked onChange={() => handleAllSubmissions()} />
+      <label>View Favorites</label>
+      <input type="radio" name="filter" onChange={() => handleFavorites()}  />
+    </div>
```
Comment (mr-ryan12) [inline src/components/AdminPage/AdminPage.js:39]: Cool, thank you. I've never used radio buttons in React before.

File: src/components/AdminPage/AdminPage.js [hunk 2]
```diff
@@ -5,30 +5,51 @@ import { useQuery } from '@apollo/client';
...
+
+    const cardsForDisplay = isFilteredSelected ? filteredSubmissions : allSubmissions
+    
+    const messageOrCards = cardsForDisplay.length ? cardsForDisplay : <p>No favorites selected yet! Please add some!</p>
+
```
Comment (mr-ryan12) [inline src/components/AdminPage/AdminPage.js:44]: Really glad this worked out!

**Review by aliroemhildt — APPROVED** [review]
This is awesome, great work Ryan and Emma! Glad we were able to get this feature working

**Review by oacortez — APPROVED** [review]
Thanks for creating this branch Emma! Great work on making the filtering for favorites to work! also creating an error message when the admin has none!

**Review by mr-ryan12 — APPROVED** [review]
Thank you Emma!

**Pull request merged**